manifests/p/ppy/osu/2024.1208.0/ppy.osu.locale.zh-CN.yaml

```
@@ -20,6 +20,7 @@ Description: |-
   一款免费的节奏游戏。节奏跃然指上。
 #Moniker: osu!
 Tags:
+  - 音游
```

```suggestion
- 音游
```

Sorry, automation error.